analytics/ga.go

```diff
@@ -42,17 +42,15 @@ type UserSession struct {
 }
 
 // NewUserSession provides a setup UserSession struct with sane defaults
-func NewUserSession() *UserSession {
+func NewUserSession(uniqueID string) *UserSession {
```

Comment:
This change and it's ancestors is a direct response to having tests fail out in a mocked environment.

https://travis-ci.org/puppetlabs/lumogon/builds/238128851?utm_source=github_status&utm_medium=notification